drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -570,6 +570,39 @@ config NET_TC_TX_COUNT
 
 endif # NETWORKING
 
+choice NRF_WIFI_NET_BUF_DATA_ALLOC_TYPE
+	prompt "Network buffer type"
```

Comment:
If this overrides the `ALLOC_TYPE` as defined under `net`, we should mention that in prompt.